src/Features/CSharp/Portable/EditAndContinue/CSharpEditAndContinueAnalyzer.cs

```diff
@@ -289,7 +291,7 @@ protected override SyntaxNode FindStatementAndPartner(SyntaxNode declarationBody
                 position = declarationBody.SpanStart;
             }
 
-            SyntaxNode node;
+            SyntaxNode? node;
```

Comment:
>? [](start = 22, length = 1)

Consider declaring as `SyntaxNode` rather than `SyntaxNode?` since it looks like most uses require it to be non-null.